README.md

```diff
@@ -1,5 +1,21 @@
-This repository serves as the collection-point, ideation hub, and process behind the [InnerSource Commons'](http://innersourcecommons.org) InnerSource Patterns--a set of proven and reviewed solutions to InnerSource problems. These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies](http://innersourcecommons.org/getting-started/). See below the list for more on what a pattern is, how to use them in your organization, and how to get involved.
+This repository contains the InnerSource Patterns collected by [innersourcecommons.org][isc-website]. These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies][isc-getting-started]. 
 
+Below you find [what a pattern is][gh-what-are-patterns], a [list of known patterns][gh-list-of-patterns], and [how to use these patterns][gh-how-to-use-patterns] in your organization.
+
+You are already using InnerSource in your company and want to share your ideas and experiences with the world? We would love to [welcome your contributions][gh-contribute]!
+
+[isc-website]: http://innersourcecommons.org
+[isc-getting-started]: http://innersourcecommons.org/getting-started/
+[gh-list-of-patterns]: https://github.com/InnerSourceCommons/InnerSourcePatterns#list-of-patterns
+[gh-what-are-patterns]: https://github.com/InnerSourceCommons/InnerSourcePatterns#what-are-inner-source-patterns
+[gh-how-to-use-patterns]: https://github.com/InnerSourceCommons/InnerSourcePatterns#how-can-you-use-inner-source-patterns
+[gh-contribute]: https://github.com/InnerSourceCommons/InnerSourcePatterns#how-to-contribute
+
+# Mission Statement
+
+We want to publish an ebook with InnerSource Patterns - a set of proven and reviewed solutions to InnerSource problems. 
+ 
+The ebook will be published on [innersourcecommons.org][isc-website]. (Disclaimer: We have not published one yet, but we will soon!)
```

Comment:
```suggestion
```

I'd drop it as it doesn't add specific information - so That way we can keep it lean?

Reply:
Sounds good.